.gitignore

```diff
@@ -1,3 +1,4 @@
 /vendor
 /build
 /.phpunit.result.cache
+/.idea/
```

Comment:
Really, this should be in your global gitignore, not in every project you ever touch. I'm not opposed to it though.

Reply:
I removed it.